app/models/medium.rb

```diff
@@ -12,14 +12,12 @@ class MissingPutFilePath < StandardError; end
   before_destroy :queue_medium_removal, unless: :external_link
 
   ALLOWED_EXPORT_CONTENT_TYPES = %w(text/csv).freeze
+  ALLOWED_CONTENT_TYPES = %w[application/json application/pdf audio/aac audio/midi audio/x-midi audio/mp3 audio/mp4 audio/x-m4a audio/mpeg audio/wav image/jpeg image/gif image/png image/tiff image/x-icon image/svg+xml text/csv text/plain video/mp4 video/mpeg].freeze
```

Comment:
perhaps add a comment and a link to the UI code to note that it's consistent and allow folks to look it up easily.

Reply:
using an array here is a O(N) at worst search operation for each media resource we want to validate. Moving to a hash would be an O(1) constant lookup.

I think the current list size is ok but something to consider if this list was to grow bigger. 

Also you could sort by most common types (image etc) to decrease the search cost but i'd probably just use a hash to speed it up and forget about it.